docs/pages/workflow/prebuild.mdx

```diff
@@ -24,7 +24,16 @@ We highly recommend using prebuild for the reasons listed in the [pitch](#pitch)
 
 ### Usage with EAS Build
 
-You can configure EAS Build to run `npx expo prebuild` before building by [using the managed preset](/build-reference/ios-builds/).
+If your project does not contain **ios** and **android** folders, then EAS Build will run Prebuild to generate your native code just before compilation. This is the default for any project started from `npx create-expo-app`.
+
+If you have a project that contains **android** and **ios** folders, then EAS Build will not run Prebuild by default in order to avoid overwriting any changes you've made to your native code.
```

Comment:
```suggestion
For a project that has **android** and **ios** directories, EAS Build will not run Prebuild by default to avoid overwriting any changes you've made to the native directories.
```